packages/web/app/forms/DischargeForm.jsx

```diff
@@ -673,9 +684,22 @@ export const DischargeForm = ({
   const { data: encounterMedications } = useEncounterMedicationQuery(encounter.id);
   const { data: ongoingPrescriptions } = usePatientOngoingPrescriptionsQuery(encounter.patientId);
 
-  const activeMedications =
-    encounterMedications?.data?.filter(medication => !medication.discontinued) || [];
-  const onGoingMedications = ongoingPrescriptions?.data?.filter(p => !p.discontinued) || [];
+  const activeMedications = (encounterMedications?.data || []).filter(
+    medication => !medication.discontinued,
+  );
+  const onGoingMedications = (ongoingPrescriptions?.data || [])
```

Comment:
"ongoing" is one word, casing is correct for `ongoingPrescriptions` 
```suggestion
  const ongoingMedications = (ongoingPrescriptions?.data || [])
```